config/locales/en.yml

```diff
@@ -1481,13 +1479,6 @@ en:
             url_hint: 'URL must start with http:// '
             url_hint_https: URL must start with http:// or https://
     unauthorized: You are not authorized to view this page.
-    user_mailer:
-      invitation:
-        ending: Thank you and have fun,
-        greeting: Your Pageflow Team
-        instruction: 'Welcome to Pageflow. Please use the following link to set your personal password. You may use your email address to sign-in. '
-        salutation: Hello %{first_name}
-        subject: Your Pageflow Invitation
```

Comment:
Please add `subect: DELETED` keys to the `invitations.de.yml` instead of deleting lines here. PRs should not touch `de.yml`/`en.yml` at all since that always leads to merge conflicts or lost changes when the files are regenerated.